terraform/modules/quorum-cluster-region/outputs.tf

```diff
@@ -37,3 +37,7 @@ output "quorum_validator_cidr_block" {
 output "quorum_observer_cidr_block" {
   value = "${lookup(var.observer_node_counts, var.aws_region, 0) > 0 ? data.template_file.quorum_observer_cidr_block.rendered : ""}"
 }
+
+output "quorum_cluster_main_route_table_id" {
```

Comment:
nit: maybe `quorum_vpc_main_route_table_id` or `region_main_route_table_id` instead here? The route table doesn't really apply to the whole cluster as this would seem to imply.

Reply:
Renamed to quorum_vpc_main_route_table_id

Reply:
This has been renamed to quorum_vpc_main_route_table_id